server/sonar-web/src/main/js/apps/issues/sidebar/AssigneeFacet.js

```diff
@@ -74,24 +76,24 @@ export default class AssigneeFacet extends React.PureComponent {
     this.props.onChange({ assigned: true, assignees: [] });
   };
 
-  handleSearch = (query: string) => {
+  handleSearch = (query /*: string */) => {
     let organization = this.props.component && this.props.component.organization;
     if (this.props.organization && !organization) {
       organization = this.props.organization.key;
     }
     return searchAssignees(query, organization);
   };
 
-  handleSelect = (assignee: string) => {
+  handleSelect = (assignee /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)